assets/css/_main-program.scss

```diff
@@ -6,47 +6,172 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  height: 76.5rem;
+  height: 104.5rem;
+
   .program-title {
     font-family: $cocogoose-family;
     color: $white-color;
     margin-top: 32.5rem;
   }
```

Comment:
- [ ] Kindly make sure that in mobile size(home page) you have the middle section individual articles have their elements organized in a horizontal row, with the image at the left, title in the center, and description at the right. And its background color should be this \`#272A31` .Take a look at the below screenshots 👇 
- Original middle section 👇 
![-3](https://user-images.githubusercontent.com/67087664/181531637-ec503eca-c219-40ee-977b-650b6b862fea.png)
- Your version 👇 
![3](https://user-images.githubusercontent.com/67087664/181532008-43a1e812-cdbe-426f-994e-00fe2950f4e1.png)